src/Tokenizer/Analyzer/CommentsAnalyzer.php

```diff
@@ -113,6 +102,24 @@ public function isBeforeStructuralElement(Tokens $tokens, int $index): bool
         return false;
     }
 
+    /**
+     * Check if comment at given index precedes return statement.
+     */
+    public function isBeforeReturn(Tokens $tokens, int $index): bool
+    {
+        if (!$tokens[$index]->isGivenKind([T_COMMENT, T_DOC_COMMENT])) {
+            throw new \InvalidArgumentException('Given index must point to a comment.');
+        }
+
+        $nextIndex = $this->getNextTokenIndex($tokens, $index);
+
+        if (null === $nextIndex || $tokens[$nextIndex]->equals('}')) {
+            return false;
+        }
+
+        return $this->isValidReturnStatement($tokens, $nextIndex);
```

Comment:
```suggestion
        return $tokens[$nextIndex]->isGivenKind(T_RETURN);
```
No need for a separate function for such a simple check.

Reply:
Agreed. fixed